po/vi.po

```diff
@@ -139,23 +139,23 @@ msgstr "Labels (nhãn)"
 
 #: src/SUMMARY.md src/control-flow-basics/blocks-and-scopes.md
 msgid "Blocks and Scopes"
-msgstr ""
+msgstr "Blocks (khối lệnh) và scopes (phạm vi)"
 
 #: src/SUMMARY.md src/control-flow-basics/blocks-and-scopes/scopes.md
 msgid "Scopes and Shadowing"
-msgstr ""
+msgstr "Scopes (phạm vi) và shadowing"
 
 #: src/SUMMARY.md src/control-flow-basics/functions.md
 msgid "Functions"
-msgstr ""
+msgstr "Hàm"
 
 #: src/SUMMARY.md src/control-flow-basics/macros.md
 msgid "Macros"
-msgstr ""
+msgstr "Macros"
 
 #: src/SUMMARY.md src/control-flow-basics/exercise.md
 msgid "Exercise: Collatz Sequence"
-msgstr ""
+msgstr "Bài tập: Chuỗi Collatz"
```

Comment:
I think this could be translated like this for consistency with other parts of the book:

```suggestion
msgstr "Thực hành: Chuỗi Collatz"
```

Reply:
Done